src/Language/Pirouette/QuasiQuoter/Syntax.hs

```diff
@@ -14,40 +15,52 @@
 -- definitions. One example program would be:
 --
 -- > -- Declares a datatype with a destructor named match_Maybe
--- > data Maybe (a : Type)
+-- > data Maybe (a : *)
```

Comment:
Any reason for changing `Type` to `*`? I tend to prefer `Type` since we already have quite a few symbols. I thing that something like: `/\ (a : (Type -> Type) -> Type) . M` is more readable than `/\ (a : (* -> *) -> *)) . M`

Reply:
The main reasons were:
- Being closer to haskell syntax and what haskellers are used to, since the general syntax aims at this
- I actually think it makes the code more readable and concise. I think the `*` makes it clear immediately we are on typevar/kind level whereas the `a : Type` looks too similar to the equivalent at termvar/type level. I feel it requires slightly more brain power to disambiguate when skimming through code and sections with nested small and big lambdas. Also it takes up less space and is more convenient to write. This is absolutely not a hill I am willing to die on though, I am fine with both :smiley:

Reply:
Rolled back to `Type` :+1: